provider/resources.go

```diff
@@ -502,9 +502,9 @@ var metadata []byte
 //
 // nolint: lll
 func Provider() tfbridge.ProviderInfo {
-	p := shimv2.NewProvider(shim.NewProvider(), shimv2.WithPlanResourceChange(func(tfResourceType string) bool {
-		return tfResourceType == "azurerm_storage_account"
-	}))
+	p := shimv2.NewProvider(shim.NewProvider(),
+		shimv2.WithPlanResourceChange(func(_ string) bool { return true }),
```

Comment:
Nit: You can drop the `_` in the function signature.
```suggestion
		shimv2.WithPlanResourceChange(func(string) bool { return true }),
```